flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FileStoreSource.java

```diff
@@ -58,15 +52,11 @@
 
     public FileStoreSource(
             FileStore fileStore,
-            RowType rowType,
-            int[] partitions,
             int[] primaryKeys,
```

Comment:
Maybe we can use `boolean keyAsRecord` instead primaryKeys?

Reply:
> Maybe we can use `boolean keyAsRecord` instead primaryKeys?

Good idea. p.s.  I prefer `recordAsKey` to `keyAsRecord`. The former is more straightforward than the latter.